akka-stream/src/main/scala/akka/stream/javadsl/Source.scala

```diff
@@ -177,6 +177,20 @@ object Source {
     new Source(scaladsl.Source.fromCompletionStage(future))
 
   /**
+   * A graph with the shape of a source logically is a source.
+   * This method makes an asynchronous graph of such shape in type with
+   * an asynchronous materialized value.
```

Comment:
First sentence can be removed as it states something general about streams rather than anything about this method. In general "this method" and "this class" is considered bad docstyle, look at the doc on the other source factory methods in this trait and mimic their way of expressing what they create.